platform/metrics/platform-metrics-reporting/pom.xml

```diff
@@ -107,6 +107,7 @@
                         <Import-Package>
                             !com.mongodb,
                             !com.sleepycat.je,
+                            org.apache.poi.*;version="[3.11,)"
```

Comment:
❓ Should we reference poi.version so that we can always use the latest version? I think this version range will work, but might be better for maintenance. I'm thinking about the potential suppressions of older versions of POI.